plugwise_usb/nodes/node.py

```diff
@@ -583,6 +588,10 @@ async def is_online(self) -> bool:
         if await self.ping_update() is None:
             _LOGGER.debug("No response to ping for %s", self.mac)
             return False
+        if not self._initialized:
+            if not await self.initialize():
+                _LOGGER.debug("Node %s failed to initialize after ping", self.mac)
+                return False
```

Comment:
Could be just the below (although I always do them this way, if the first argument catches, the second will never be evaluated so you can combine them):

```
if not self._initialized and not await self.initialize():
    _LOGGER.debug("Node %s failed to initialize after ping", self.mac)
    return False
    ```